### PR TITLE
Feat/validate native attributes

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: packages/form-validations/src/FormValidations.ts [hunk 1]
```diff
@@ -95,15 +127,15 @@ class FormValidations {
   /**
    * Checks whether the form/input elements are valid
    *
-   * @param input => input element(s) or a jquery selector, null for full form validation
```
Comment (crishpeen) [inline packages/form-validations/src/FormValidations.ts:98]: 😁

File: packages/form-validations/src/lang.ts [hunk 2]
```diff
@@ -4,5 +4,16 @@ export const lang: Languages = {
...
     required: 'This field is required',
     default: 'Please enter a correct value',
+    email: 'This field requires a valid e-mail address',
+    number: 'This field requires a number',
+    integer: 'This field requires an integer value',
+    url: 'This field requires a valid website URL',
+    tel: 'This field requires a valid telephone number',
+    maxlength: `This fields length must be < ${1}`,
+    minlength: `This fields length must be > ${1}`,
+    min: `Minimum value for this field is ${1}`,
```
Comment (crishpeen) [inline packages/form-validations/src/lang.ts:14]: The replace doesn't work here. Please fix it

File: packages/form-validations/README.md
[the remark this answers is not in the document]
Comment (literat) [reply packages/form-validations/README.md:1]: Fixed in https://github.com/lmc-eu/spirit-design-system/pull/830

File: packages/form-validations/README.md [hunk 3]
```diff
@@ -109,4 +109,21 @@ _Constructor_
...
 | config    | [See above](#default-config) | ✕         | The config object                                   |
 | live      | `true`                       | ✕         | Whether FormValidations should validate as you type |
+
+**FormValidations.validate(inputs, silent)**
+
+_Validate the form or field(s)_
+
+| Parameter | Default | Required? | Description                                                                                                                                                             |
+| --------- | ------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| inputs    | -       | ✕         | When not given, full form is validated. inputs can be one DOM element or a collection of DOM elements returned by `document.getElement...`, `document.querySelector...` |
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/form-validations/README.md]: Ok, fixed.

File: packages/form-validations/src/lang.ts [hunk 4]
```diff
@@ -4,5 +4,16 @@ export const lang: Languages = {
   en: {
     required: 'This field is required',
     default: 'Please enter a correct value',
+    email: 'This field requires a valid e-mail address',
+    number: 'This field requires a number',
+    integer: 'This field requires an integer value',
+    url: 'This field requires a valid website URL',
+    tel: 'This field requires a valid telephone number',
+    maxlength: `This fields length must be < ${1}`,
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/form-validations/src/lang.ts]: Ok, fixed.

File: packages/form-validations/src/lang.ts (hunk 2 above)
Comment (literat) [reply packages/form-validations/src/lang.ts]: Fixed.

File: packages/form-validations/src/types.ts [hunk 5]
```diff
@@ -6,23 +6,12 @@ export interface Languages {
...
 export type Params = Record<string, string[] | string> | never[];
 
 export type Validator = {
-  fn: (value: string) => boolean | number;
-  priority: number;
-  halt: boolean;
+  fn: (...args: string[]) => boolean | number;
+  priority?: number;
+  halt?: boolean;
   msg?: string | ((message: string, params: Params) => string);
```
Comment (tomassychra) [inline packages/form-validations/src/types.ts:15]: message?

File: packages/form-validations/README.md [hunk 6]
```diff
@@ -109,4 +110,24 @@ _Constructor_
...
 | config    | [See above](#default-config) | ✕         | The config object                                   |
 | live      | `true`                       | ✕         | Whether FormValidations should validate as you type |
+
+**FormValidations.validate(inputs, silent)**
+
+_Validate the form or field(s)_
+
+| Parameter | Default | Required? | Description                                                                                                                                                                          |
+| --------- | ------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| inputs    | -       | ✕         | When not given, the entire form is validated. Inputs can either be one DOM element or a collection of DOM elements returned by `document.getElement...`, `document.querySelector...` |
```
Comment (crishpeen) [inline packages/form-validations/README.md:120]:
```suggestion
| inputs    | -       | ✕         | When not given, the entire form is validated. Inputs can either be one DOM element or a collection of DOM elements returned by `document.getElement…`, `document.querySelector…` |
```

File: packages/form-validations/src/lang.ts [hunk 7]
```diff
@@ -1,8 +1,20 @@
...
   en: {
     required: 'This field is required',
     default: 'Please enter a correct value',
+    email: 'This field requires a valid e-mail address',
+    number: 'This field requires a number',
+    integer: 'This field requires an integer value',
+    url: 'This field requires a valid website URL',
+    tel: 'This field requires a valid telephone number',
+    maxlength: 'This fields length must be <= ${1}',
+    minlength: 'This fields length must be => ${1}',
```
Comment (crishpeen) [inline packages/form-validations/src/lang.ts:14]: this creates an arrow in my browser 😁 
how about we use the correct symbols? ≤ ≥

**Review by tomassychra — APPROVED** [review]
GJ

File: packages/form-validations/src/lang.ts (hunk 7 above)
Comment (literat) [reply packages/form-validations/src/lang.ts]: Fixed.

File: packages/form-validations/README.md (hunk 6 above)
Comment (literat) [reply packages/form-validations/README.md]: Fixed.

File: packages/form-validations/src/types.ts (hunk 5 above)
Comment (literat) [reply packages/form-validations/src/types.ts:15]: Yes, you are right. I will refactor this, and same the `fn` in the last PR before merge.

**Pull request merged**